piper/ratbagd.py

```diff
@@ -205,15 +219,22 @@ class Ratbagd(_RatbagdDBus):
 
     def __init__(self):
         _RatbagdDBus.__init__(self, "Manager", None)
+        result = self._get_dbus_property("Devices")
+        self._devices = [RatbagdDevice(objpath) for objpath in result]
+
+    def _on_properties_changed(self, proxy, changed_props, invalidated_props):
+        if "Devices" in changed_props.keys():
+            for prop in changed_props["Devices"]:
+                index = self._find_object_with_path(self._devices, prop)
```

Comment:
`So if __eq__ is implemented to return true if self.objpath == that_string, then it should work.`

That will work yes, but I'm not sure about it either. Let's leave this as-is for now; the removal is a bit more lines of code than it can be but at least the complexity is in an obvious place; I don't like hiding it away in `__eq__`.